modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-installing-operator-from-operatorhub-using-cli-{context}']
+= Installing from the OperatorHub using the CLI
+
+Instead of using the {product-title} web console, you can install an Operator
+from the OperatorHub using the CLI. Use the `oc` command as follows to install
+(subscribe to) an Operator for a selected namespace.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin`
+permissions.
+
+.Procedure
+
+. View the list of Operators available to the cluster from the OperatorHub:
++
+----
+$ oc get packagemanifests -n openshift-marketplace
+----
+
+. To identify the Operators to install, create a CatalogSourceConfig object YAML
+file (for example, `csc.cr.yaml`) containing information similar to the
+following:
+
++
+.Example CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "example"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "global" <1>
+  packages: "myoperator,another-operator" <2>
+----
+<1> Set the `targetNamespace` to identify the namespace where you want the
+Operator to be available.
+<2> Set `packages` to a comma-separated list of Operators to which you want to
+subscribe.
+
+. Install the Operator to the selected namespace:
++
+----
+$ oc apply -f csc.cr.yaml
+----
+
+. Later, if you want to install more Operators, update your CatalogSourceConfig
```

Comment:
This won't install operators either.

Reply:
@adellape @chrisnegus In the CLI installing an operator involves, creating or updating a `CatalogSource` object followed by adding a `Subscription` object. The UI does this for you behind the scenes and the user thinks it is one step.

Reply:
Added some context and a "NOTE" box to the beginning of http://file.rdu.redhat.com/~adellape/022219/hub_cli_edits/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli-olm-adding-operators-to-a-cluster.